spine_aws_common/web_application.py

```diff
@@ -0,0 +1,86 @@
+"""
+Base Web Lambda application
+"""
+from typing import List, Callable, Optional, Dict
+import re
+import json
+
+from aws_lambda_powertools.event_handler.api_gateway import (
+    Route,
+    ResponseBuilder,
+    Response,
+)
+
+from spine_aws_common.lambda_application import LambdaApplication
+
+
+class WebApplication(LambdaApplication):
+    """
+    Base class for Web Lambda applications
+    """
+
+    def __init__(self, additional_log_config=None, load_ssm_params=False):
+        super().__init__(
+            additional_log_config=additional_log_config, load_ssm_params=load_ssm_params
+        )
+        self._routes: List[Route] = []
+
+    def initialise(self):
+        self._routes: List[Route] = []
```

Comment:
Does these need to be here in the base class? re-adding the empty `_routes` seems redundant